### PR TITLE
Feature2

### PR DESCRIPTION
**Pull request by Buyaki01**
## To do list functionality
In this feature:
 - The user will also be able to mark task completion by selecting the corresponding checkbox (or undo it by unchecking the checkbox).
 - The updated tasks list will be stored in local storage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,47 @@
...
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- Webpack
+
+##Live Demo
+![live demo link](https://competent-swirles-bcd6c1.netlify.app)
```
Comment (SajjadAhmad14) [inline README.md]: Please add an active live demo link or remove this option from the README.

File: README.md [hunk 2]
```diff
@@ -1 +1,47 @@
...
+
+## Getting Started
+
```
Comment (SajjadAhmad14) [inline README.md]: The project looks awesome! But README should be professional and clear for users. Please add `npm install` and `npm start` commands so a non-technical user can set the project up locally.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Ritta, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
PR has a title and summary. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by mikethreels — APPROVED** [review]
Hi Ritta, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**